File: Java Script - Introduction to Programming/7.2. Complex Loops - Exam Problems/02. Magic Numbers.js
function magicNumbers(magicNum) {
    let resultList = []

    for (let one = 1; one <= 9; one++) {
        for (let two = 1; two <= 9; two++) {
            for (let three = 1; three <= 9; three++) {
                for (let four = 1; four <= 9; four++) {
                    for (let five = 1; five <= 9; five++) {
                        for (let six = 1; six <= 9; six++) {
                            if (one * two * three * four * five * six == magicNum) {
                                resultList.push(`${one}${two}${three}${four}${five}${six}`)
                            }
                        }
                    }
                }
            }
        }
    }

    return resultList.join(' ')
}

// console.log(magicNumbers(2))

File: Pyrhon - Introduction to Programming/7.2. Complex Loops - Exam Problems/02. Magic Numbers.py
magic_num = int(input())
result_list = []

for one in range(9):
    for two in range(9):
        for three in range(9):
            for four in range(9):
                for five in range(9):
                    for six in range(9):
                        if one * two * three * four * five * six == magic_num:
                            result_list.append(f'{one}{two}{three}{four}{five}{six}')

print(' '.join(result_list))


'''
2
#111112 111121 111211 112111 121111 211111
'''
